tests/serverpod_test_server/test/serialization_legacy_test.dart

```diff
@@ -291,6 +294,12 @@ void main() {
           UuidValue.fromString('a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11'),
           null,
         ],
+        aBigInt: BigInt.one,
+        aBigIntList: [BigInt.one, BigInt.two],
+        aListWithNullableBigInts: [BigInt.one, null],
+        aNullableBigIntList: [BigInt.one],
+        aNullableBigInt: BigInt.one,
+        aNullableListWithNullableBigInts: [BigInt.one, null],
```

Comment:
Suggestion: I think you can remove this test and just disregard anything in here. These are legacy tests that we have not spent time on migrating yet, but we are consciously not adding new things in here! Instead all the tests inside `serialization_test.dart` should provide the full coverage for the types.

Reply:
Reverted the addition to the legacy tests.